From cc551cb0423a1de1e969174ad12a0b53191bab9b Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Thu, 19 Jan 2017 13:34:51 +0100
Subject: [PATCH 1/1] standalone: work around borked dependencies in acl

For any reason acl decides do build getfacl and setfacl on `make [...]
install-lib install-dev`, which may fail. Make sure to not fail by
installing from libacl and include subdirectories.

Signed-off-by: Christian Hesse <mail@eworm.de>
---
 GNUmakefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/GNUmakefile b/GNUmakefile
index 1a9bfa0..712a35d 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -201,7 +201,8 @@ dependency/build/libacl-build: dependency/build/libacl-configure
 	touch $@
 
 dependency/build/libacl-install: dependency/build/libacl-build
-	$(MAKE) -C $(dir $<)/$(LIBACL) DESTDIR=$(DEPS_ROOT) install-lib install-dev
+	$(MAKE) -C $(dir $<)/$(LIBACL)/libacl DESTDIR=$(DEPS_ROOT) install-dev
+	$(MAKE) -C $(dir $<)/$(LIBACL)/include DESTDIR=$(DEPS_ROOT) install-dev
 	touch $@
 
 # COMMON
-- 
2.11.0

